test/jdk/java/util/BitSet/ImmutableBitSet.java

```diff
@@ -50,28 +51,38 @@ void empty() {
     void negativeIndex() {
         BitSet bs = new BitSet();
         IntPredicate ibs = ImmutableBitSetPredicate.of(bs);
-        assertThrows(IndexOutOfBoundsException.class, () -> {
-            ibs.test(-1);
-        });
+        assertFalse(ibs.test(-1));
+        assertFalse(ibs.test(Integer.MIN_VALUE));
+
+        bs = new BitSet(1024);
+        ibs = ImmutableBitSetPredicate.of(bs);
+        assertFalse(ibs.test(-1));
+        assertFalse(ibs.test(Integer.MIN_VALUE));
     }
 
     @Test
     void basic() {
-        BitSet bs = createReference(147);
+        IntStream.of(0, 16, 143, 4711).forEach(k -> basic(k));
```

Comment:
A test on either side of the boundary (128) in the implementation might be useful.

Reply:
Thanks for reviewing. I added a few cases and made the `BitSet` creation more explicit w.r.t. size.